### ISSUE TITLE
Gradebook not updating when changing and saving assignment grade

### ISSUE THREAD

**Issue by rrobidoux** [RunestoneInteractive/RunestoneServer#1710]
Course: Summer-2021-MobileCSP-PD

Pages: Gradebook

User: mobilecsp-team

TL;DR The Gradebook does not update student percent scores when making grade changes in the gradebook. It will only update if the teacher opens the Grading tab, chooses the assignment and clicks "Autograde and Display Total". 

Finding the issue...
Open the Gradebook
Click on a student's grade for an assignment with at least 1 manually graded question
Click one of the manually graded questions
Type in a grade and click "Save Grade"
The student's score is updated and is correct (In this case moving from 3/8 to 4/8)
Open the Gradebook again
The student's percentage has not been updated but the updated score is present when clicking on the assignment
<img width="746" alt="clickingSaveGrade" src="https://user-images.githubusercontent.com/30819358/126812012-98ae7ed8-4f9e-4627-82b8-0f5045a12b44.png">
<img width="730" alt="scoreIsAdjusted" src="https://user-images.githubusercontent.com/30819358/126812057-56c96f2e-ff4a-4db5-a798-80fe85e1ea50.png">
<img width="103" alt="studentUnchangedScore" src="https://user-images.githubusercontent.com/30819358/126811961-5ac6f19c-2801-4801-8d0c-f6f5816e1932.png">

Javascript Errors: None appeared in the console
<img width="617" alt="openRunestoneAssignment_console" src="https://user-images.githubusercontent.com/30819358/126811909-020d9c03-a468-4cfe-afdf-0a6a10dedfde.png">

**Comment by bnmnetp** [comment]
Fixed recently...
